src/guides/v2.3/install-gde/bk-install-guide.md

```diff
@@ -13,12 +15,10 @@ We have some resources here to help get you started using the eCommerce platform
 
 ## How to get the Magento software {#install-get-software}
 
-Check the availability of exciting new features and releases, and learn how you can obtain them, on our [Magento 2.3 product availability page](https://devdocs.magento.com/release/availability.html).
+Check the availability of exciting new features and releases, and learn how you can obtain them, on our [Magento product availability page](https://devdocs.magento.com/release/availability.html).
```

Comment:
A nit pick for sure, but can we get rid of `exciting`?

Reply:
No problem